CMakeLists.txt

```diff
@@ -70,6 +70,9 @@ endif()
 #------------------------------------------------------------------------------
 # modules and definitions
 #------------------------------------------------------------------------------
+include(cmake/utils.cmake)
```

Comment:
I'm not sure that this utils.cmake can be include here (before the python stuff from lines 81-100). Because in utils.cmake it uses the var "Python_EXECUTABLE" at [line 70](https://github.com/Autodesk/maya-usd/blob/dev/cmake/utils.cmake#L70). I'm not enough of a cmake expert to know if that var is replaced during the include of utils.cmake or only when the function _mayaUsd_find_python_module_ is called? If the latter then we should be okay.

Reply:
It'll only do the lookup when the function is called.  So as long as the Python executable is defined before Line 102 (`include(cmake/jinja.cmake)`) it should be fine.

Reply:
I wonder if we should add a message(FATAL_ERROR, ...) to confirm that Python_EXECUTABLE is valid in that function?